geometry_msgs/Pose  brick_pose
geometry_msgs/Pose  feeder_pose
std_msgs/String     brick_type
std_msgs/Bool     is_placed
